e2e/e2e_test.go

```diff
@@ -188,14 +199,13 @@ func sendHbarsToBridgeAccount(setup *setup.Setup, memo string) (*hedera.Transact
 	return &res, err
 }
 
-func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.TransactionResponse, t *testing.T) string {
+func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.TransactionResponse, expectedSignaturesCount int, expectedEthTxMessageCount int, t *testing.T) string {
```

Comment:
Do we need to pass `expectedSignaturesCount`?

Reply:
This is needed in order to generalize the verification of different scenarios. With a format such as address-0-0 we are waiting for no ethTxMessage, therefore we pass a 0 at the assertion.